src/Core/Content/Product/SalesChannel/Listing/Filter.php

```diff
@@ -54,8 +32,7 @@ public function __construct(
         bool $filtered,
         array $aggregations,
         DALFilter $filter,
-        /** @deprecated tag:v6.7.0 - Will be natively typed */
-        protected $values,
+        protected mixed $values,
```

Comment:
```suggestion
        protected int|float|string|bool|array|null $values,
```

please use dedicated types, as mixed includes every possible type, like classes